Index.html

```diff
@@ -18,95 +18,208 @@
+                                <p>Codekit</p>
+                            </li>
+                            <li>
+                                <p>Github</p>
+                            </li>
+                            <li>
+                                <p>Codepen</p>
+                            </li>
+                            <li>
+                                <p>Gitlab</p>
+                            </li>
+                            <li>
+                                <p>Terminal</p>
+                            </li>
+                        </ul>
+                    </section>
+                </article>
+                <div class="vector">
+                    <img src="./images/Vector4.png" alt="vector">
+                </div>
+                <div class="socialContent">
+                    <ul>
+                        <li><a href="#"><img src="./icons/github.png" alt="github icon"></a></li>
+                        <li><a href="#"><img src="./icons/linkedIn.png" alt="linkedIn icon"></a></li>
+                        <li><a href="#"><img src="./icons/angelist.png" alt="angelist icon"></a></li>
+                        <li><a href="#"><img src="./icons/twitter.png" alt="twitter icon"></a></li>
+                        <li><a href="#"><img src="./icons/medium.png" alt="medium icon"></a></li>
+                    </ul>
+                    <div class="element">
+                        <img src="./images/arrow.png" alt="arrow" >
+                        <p class="work">Work</p>
+                    </div>
+                </div>
 
-                </ul>
             </section>
-            <section class="box">
-                <h2>Skills</h2>
-                <ul>
-                    <li>
-                        <p>Codekit</p>
-                    </li>
-                    <li>
-                        <p>Github</p>
-                    </li>
-                    <li>
-                        <p>Codepen</p>
-                    </li>
-                    <li>
-                        <p>Gitlab</p>
-                    </li>
-                    <li>
-                        <p>Terminal</p>
-                    </li>
-                </ul>
-            </section>
-        </article>
-        <div class="vector">
-            <img src="./images/Vector4.png" alt="vector">
         </div>
-        <div class="socialContent">
-            <ul>
-                <li><a href="#"><img src="./icons/github.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="./icons/linkedIn.png" alt="linkedIn icon"></a></li>
-                <li><a href="#"><img src="./icons/angelist.png" alt="angelist icon"></a></li>
-                <li><a href="#"><img src="./icons/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="#"><img src="./icons/medium.png" alt="medium icon"></a></li>
-            </ul>
+        <div class="grid-item recent-work">
+            <p>MY RECENT WORK</p>
```

Comment:
- [ ] Kindly use `h2-6` instead of `p` because this is a headline so it's more semantic to use header tags.
- [ ] Kindly give it some `padding-left` to be like the Figma design.

![image](https://user-images.githubusercontent.com/35535130/154598526-fda2711b-adbd-485d-a2c8-ed0ce83dbad0.png)